src/ru/umeta/zharvester/SQLiteJDBC.java

```diff
@@ -5,36 +5,44 @@
 public class SQLiteJDBC{
 
     public static boolean db_check(Integer hash, String location) {
```

Comment:
method, variables, fields. parameters should be in camel case starting with lowercase letter, e.g. "dbCheck".